src/lib/create-utils.js

```diff
@@ -85,6 +86,33 @@ exports.enhanceReadmeWithLocalSuffix = async (readmePath, manifestPromise) => {
     }
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actFolderDir
+ */
+exports.createInputFromSchema = async (actFolderDir) => {
```

Comment:
The name doesn't match the purpose of the function, I'd go with sth like `createInputFileFromInputSchema` or similar.